tools/test-js-with-node

```diff
@@ -63,6 +67,44 @@ if not options.force:
         print('If you really know what you are doing, use --force to run anyway.')
         sys.exit(1)
 
+NODE_COVERAGE_PATH = 'var/node-coverage/coverage.json'
+
+if options.check_enforced:
+    coverage_json = None
+    print()
+    print("=============================================================================")
+    print("Checking for fully covered files that are not enforced yet...")
+    print()
+    try:
+        with open(NODE_COVERAGE_PATH, 'r') as f:
+            coverage_json = ujson.load(f)
+    except IOError:
+        print(NODE_COVERAGE_PATH + " doesn't exist. Try running `test-js-with-node --coverage` first.")
+        raise
```

Comment:
I feel like it would be simpler if this option just implied `--coverage` and ran the tests for you; they're pretty fast and then you don't need to worry about caching being inaccurate.